### PR TITLE
Custom chart downloading as PNG

### PR DESCRIPTION
**Pull request by Th3-Hero**
- Removed Millimechs
- Added dom-to-image and file-saver
- Added ability to download chart as PNG

### CODE REVIEW

File: src/components/time-on-target-chart/TimeOnTargetChart.svelte [hunk 1]
```diff
@@ -31,63 +31,65 @@
...
+    
+            <h3 class="mt-2 mb-0">Accuracy</h3>
+            <select class="dropdown-select mt-1" bind:value={accuracy}>
+                <option value="100">100%</option>
+                <option value="75">75%</option>
+                <option value="50">50%</option>
+            </select>
+    
+            <button class="download-button" on:click={chartToImage}>Download Chart <br> 
+                                                                        as PNG</button>
```
Comment (k-seth) [inline src/components/time-on-target-chart/TimeOnTargetChart.svelte]: Why the new line here? Just keep in on the same line...

There is also no need for the blank line under this

File: src/components/time-on-target-chart/TimeOnTargetChart.svelte [hunk 2]
```diff
@@ -31,63 +31,65 @@
...
+                <div></div>
+            {/if}
+        </div>
 
-            <div class="graphic mt-4 is-justify-content-center">
-                {#each THARGOID_TYPES as interceptor}
-                    <InterceptorData bind:accuracy bind:ammo interceptor={interceptor.name}/>
-                {/each}
-            </div>
+        <div class="graphic mt-4 is-justify-content-center">
```
Comment (k-seth) [inline src/components/time-on-target-chart/TimeOnTargetChart.svelte:87]: Looks like the line endings were changing types here potentially? We should make sure that there is a consistent setting here so there aren't constant unnecessary changes.

File: src/components/time-on-target-chart/TimeOnTargetChart.svelte [hunk 3]
```diff
@@ -31,63 +31,65 @@
...
-
-        <label class="toggle-mechs">
-            <input type=checkbox bind:checked={toggleMm}>
-            Toggle Millimechs
-        </label>
-
+    <div class="chart-settings">
+        <div class="mb-3 ml-2">
+            <h1 class="mt-1 mb-0 has-text-centered">Create a Chart</h1>
+
```
Comment (k-seth) [inline src/components/time-on-target-chart/TimeOnTargetChart.svelte]: Several of these blank lines have extra white (4 spaces). Please remove them

**Review by k-seth — CHANGES_REQUESTED** [review]

File: src/components/time-on-target-chart/TimeOnTargetChart.svelte (hunk 1 above)
Comment (Th3-Hero) [reply src/components/time-on-target-chart/TimeOnTargetChart.svelte]: Fixed

File: src/components/time-on-target-chart/TimeOnTargetChart.svelte (hunk 3 above)
Comment (Th3-Hero) [reply src/components/time-on-target-chart/TimeOnTargetChart.svelte]: Fixed

**Review by k-seth — APPROVED** [review]
